include/nuttx/sched.h

```diff
@@ -632,6 +633,12 @@ struct tcb_s
 
   FAR void *waitobj;                     /* Object thread waiting on        */
 
+  /* Schedule Event Group support *******************************************/
+
+#ifdef CONFIG_SCHED_EVENTS
+  event_mask_t events;                   /* Schedule Events                 */
```

Comment:
how can we set value in tcb->events

Reply:
Through the `nxevent_wait()` interface, the waiting event will be set to the current tcb